src/components/VectorInput/index.js

```diff
@@ -203,7 +203,9 @@ class VectorInput extends Element {
     /* eslint accessor-pairs: 0 */
     set values(values) {
         // create an array for each dimension (e.g. one array for x one for y one for z)
-        values = this._inputs.map((_, i) => values.map((arr) => arr[i]));
+        values = this._inputs.map((_, i) => values.map((arr) => {
```

Comment:
Consider moving the comment above to the block below and write a new comment specific to this block.

Reply:
I'm not sure I understand :D